features/step_definitions/user_steps.rb

```diff
@@ -45,6 +45,11 @@
 
 Given /^I am logged in as "([^\"]*)" with password "([^\"]*)"$/ do |login, password|
   step("I am logged out")
+  step(%{I am logged in as "#{login}" with password "#{password}" no logout step})
```

Comment:
I am trying to be dry. I needed a version which didn't log out ( I may be able to refactor this, now with more sleep ).